canonical/lxd-ui
# fix(errors) use titled notifications for all error notifications

## Done

- changed the failure function so it expects a title and the error, optional message and actions
- adjusted all failures, so we have concise and consistent titles
- this should make it easier for the user to read the actual error message (in the notification body). Before we'd mix our message with the API error message in the body, making it hard to gasp the notification.

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described in the [Readme](https://github.com/canonical/lxd-ui#setting-up-for-development).
2. Perform the following QA steps:
    - check the errors that are outputted
